skimage/measure/fit.py

```diff
@@ -434,6 +434,11 @@ def estimate(self, data):
         except np.linalg.LinAlgError:  # LinAlgError: Singular matrix
             return False
 
+        # For some reason the above doesn't catch the error on linux 32 bit
+        # https://stackoverflow.com/questions/13249108/efficient-pythonic-check-for-singular-matrix
+        if np.linalg.cond(M) > 1 / np.finfo(np.double).eps:
```

Comment:
This check can be done before the inversion is attempted?  It simplifies the logic a bit.

Reply:
I'm not sure. in fact, when I computed `cond(M)` in the debugger with `import pdb; pdb.set_trace()`, the condition number was already `Inf`. I don't really want to cause other overflow errors or "old numpy specific errors"

From the docs
> eps | (float) The smallest representable positive number such that 1.0 + eps != 1.0.  Type of eps is an appropriate floating point type.

They don't seem to have a eps^-1. Thought I could do
`eps**-1` if you like.